packages/sn1per/PKGBUILD

```diff
@@ -23,6 +23,7 @@ depends=('amap' 'arachni' 'bash' 'bind' 'brutex' 'cisco-torch' 'curl' 'jq'
          'testssl.sh' 'theharvester' 'unzip' 'w3af' 'wapiti' 'whatweb'
          'whois' 'wpscan' 'xprobe2' 'xsstracer' 'xdg-utils')
 makedepends=('git')
+optdepends=('firefox')
```

Comment:
Can you add the short description: https://wiki.archlinux.org/title/PKGBUILD#optdepends

Reply:
Added.